lib/siwapp_web/live/customer_live/index.html.heex

```diff
@@ -0,0 +1,30 @@
+<h1> <%= @page_title %> </h1>
+  <table class="table is-responsive is-bordered is-striped is-hoverable is-fullwidth">
+    <thead>
+      <tr>
+        <th class="no-wrap" title="Customer name"> Customer name </th>
+        <th class="no-wrap" title="Identification"> Identification </th>
+        <th class="text-right" title="Due"> Due </th>
+        <th class="text-right" title="Total"> Total </th>
+        <th class="text-right" title=" "></th>
+      </tr>
+    </thead>
+
+    <tbody role="infinite-content">
+      <%= for customer <- @customers do%>
+        <%=if customer.active do %>
+          <tr>
+            <td class="no-wrap"> <%= live_patch customer.name, to: Routes.customer_edit_path(@socket, :edit, customer.id) %> </td>
```

Comment:
why live_patch? Please read docs

Reply:
I used live_patch because I understood that maybe it's not necessary to shut down the whole view and we can only update the differences. Quoting josevalim from [this ](https://elixirforum.com/t/concrete-examples-of-when-to-use-live-patch-live-redirect-push-redirect-push-patch/30751/9) elixir forum: "live_redirect/2 and push_redirect/2 reloads the LiveView but
keeps the current layout

live_patch/2 and push_patch/2 updates the current LiveView and
sends only the minimal diff"

Reply:
Also, the implementation of edit already contains the handle_params function so I thought this would be a good option. Maybe you are looking for live_redirect, I didn't see it so clear and what I'm seeing in forums so far, some people see the ambiguity that I'm exposing

Reply:
Edit e index son diferentes liveview, de ahí que no se pueda usar live_patch. Te siguen patinando los conceptos. Dale caña al libro.